lib/runtime/test_helpers.go

```diff
@@ -145,7 +147,7 @@ func generateEd25519Signatures(t *testing.T, n int) []*Signature {
 // GenerateRuntimeWasmFile generates all runtime wasm files.
 func GenerateRuntimeWasmFile() ([]string, error) {
 	var wasmFilePaths []string
-	runtimes := []string{HOST_API_TEST_RUNTIME, POLKADOT_RUNTIME, NODE_RUNTIME}
+	runtimes := []string{HOST_API_TEST_RUNTIME, POLKADOT_RUNTIME, NODE_RUNTIME, DEV_RUNTIME}
```

Comment:
nit: make this a a `const` on the package level?

Reply:
done